docs/headless-cms/docs/06-02-clusterdocstopic.md

```diff
@@ -0,0 +1,86 @@
+---
+title: ClusterDocsTopic
+type: Custom Resource
+---
+
+The `clusterdocstopics.cms.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define an orchestrator that creates ClusterAsset CRs for a specific asset type. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd clusterdocstopics.cms.kyma-project.io -o yaml
+```
+
+## Sample custom resource
```

Comment:
please make more meaningful example that is close to reality
```
apiVersion: cms.kyma-project.io/v1alpha1
kind: ClusterDocsTopic
metadata:
  name: service-catalog
  labels:
    cms.kyma-project.io/viewContext: docs-ui
    cms.kyma-project.io/order: "2"
    cms.kyma-project.io/groupName: Components
spec:
  displayName: "Service Catalog"
  description: "Service Catalog documentation"
  sources:
    markdown:
      mode: package
      url: https://github.com/kyma-project/kyma/archive/0.8.1.zip
      filter: ^kyma-0.8.1/docs/service-catalog/docs
```

Reply:
Could you provide one that is concise and contains all described parameters?

Reply:
Ok, I added the sample provided by M.Hudy.